src/components/checkbox/checkbox.ts

```diff
@@ -150,7 +153,9 @@ export class MdCheckbox implements AfterContentInit, ControlValueAccessor {
 
       // Only fire a change event if this isn't the first time the checked property is ever set.
       if (this._isInitialized) {
-        this._emitChangeEvent();
+        if (document.activeElement === this._inputElement.nativeElement) {
```

Comment:
I think using `document.activeElement` _might_ cause problems with Angular Universal. Since the checkbox already keeps track of focus with `hasFocus`, just use that?